Ryujinx.Graphics.Gpu/Image/TextureGroupHandle.cs

```diff
@@ -159,6 +159,46 @@ public void RecalculateOverlaps(TextureGroup group, List<Texture> views)
             }
         }
 
+        /// <summary>
+        /// Adds a single texture view as an overlap if its range overlaps.
+        /// </summary>
+        /// <param name="offset">The offset of the view in the group</param>
+        /// <param name="view">The texture to add as an overlap</param>
+        public void AddOverlap(int offset, Texture view)
+        {
+            // Overlaps can be accessed from the memory tracking signal handler, so access must be atomic.
+
+            int endOffset = Offset + Size;
+
+            if (offset < endOffset && Offset < offset + (int)view.Size)
```

Comment:
```suggestion
            if (OverlapsWith(offset, (int)view.Size))
```
And then `int endOffset = Offset + Size;` can be removed.